cmd/brew-setup-nginx-conf

```diff
@@ -102,10 +101,4 @@ unless started_services
   unless system "brew services restart nginx >/dev/null"
     abort "Error: failed to (re)start nginx!"
   end
-  unless system "sudo -n true >/dev/null"
-    puts "Asking for your password to restart launch_socket_server:"
-  end
-  unless system "sudo brew services restart launch_socket_server >/dev/null"
```

Comment:
We could guard this section by grepping brew services list to see if launch_socket_server is running, and start it if not. That way, sudo wouldn't be needed on every run.